.github/dependabot.yml

```diff
@@ -54,17 +48,6 @@ updates:
     commit-message:
       prefix: "githubrelease"
       include: "scope"
-  - package-ecosystem: "docker"
-    directory: "/development/kyma-ci-force-bot/ciforcebot"
```

Comment:
Why have you removed ciforcebot?

Reply:
Ciforcebot was never used. It's a leftover from POC.